cloudify_cli/commands/deployments.py

```diff
@@ -68,6 +73,67 @@ def deployments():
     pass
 
 
+def _print_single_update(deployment_update_dict,
+                         explicit_reinstall=None,
+                         preview=False,
+                         skip_install=False,
+                         skip_uninstall=False,
+                         skip_reinstall=False):
+    if explicit_reinstall is None:
+        explicit_reinstall = []
+    columns = DEPLOYMENT_UPDATE_COLUMNS
+    if preview:
+        columns = [c for c in columns if c not in NON_PREVIEW_COLUMNS]
```

Comment:
This being a double negative isn't as nice as it could be- maybe we could reverse the column listing above to make it a set with some elements removed and then we wouldn't need to do this?